_posts/2022-02-05-frenchaward.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "An Open Source Software Award for scikit-learn"
+date: January 8, 2022
+categories:
+  - Events
+tags:
+  - Open Source
+  - Award
+  - Community
+  - French government
```

Comment:
```suggestion
  - Open Source
  - Community
```

Reply:
@francoisgoupil 
I think it's good to limit to a couple of tags that we already have.